### DISCUSSION TITLE
Is there a way to see tests?

### DISCUSSION THREAD

**Discussion by AbsoluteGoof** [codewars/codewars.com#2749] [Q&A]
When I test or attempt, it tells me things have passed, but not what they were. Fails are the same way, I want to know what is in the test so I know what's going wrong, but I don't know how to check to see what tests are failing. Any solutions?

**Comment by EloiseRosen** [comment]
You can print out the input.
For example in python you could do:
```
def func(x):
    print(x)
    [rest of your code]
 ```

**Comment by hobovsky** [comment]
Ideal solution would be to make authors and translators finally realize that it's really helpful when a failed test is accompanied by at least a little bit of a context other than `"should work"`.
It's not easy, resistance is strong, but slowly getting there.

**Reply by Harold2017** [reply]
> Ideal solution would be to make authors and translators finally realize that it's really helpful when a failed test is accompanied by at least a little bit of a context other than `"should work"`. It's not easy, resistance is strong, but slowly getting there.

Especially in C/C++, some random tests given UB and it is quite hard to guess what happened if known nothing about the inputs...
